solarforecastarbiter/reports/templates/html/body.html

```diff
@@ -126,6 +127,14 @@ <h1 id="report-title">{{ report_name }}</h1>
 
 {% block errors %}
 {% set messages = (report.raw_report.messages | map(attribute="message") | list) + (templating_messages | default([]))%}
+
+{% if report_outages | length < report.outages | length %}
+  {{ messages.append(
+    "Outages have been added since the report was last computed. "
+    "Recompute to use the new outages.")
+   }}
```

Comment:
report outages could also be removed, right? So should we test for equality and then say modified instead of added? Though I suppose we'd still be in trouble if the same number were removed as added. Could address that in follow up work.

Reply:
I don't think we can actually detect removal. Since `report.outages` would have been modified. Maybe the better way to do all this is to add an `outages` property to the `RawReport` object and check for equality and forget including the `created_at` and `modified_at` properties for `TimePeriod`?